examples/sampling/graphbolt/lightning/node_classification.py

```diff
@@ -133,7 +133,7 @@ def configure_optimizers(self):
 
 
 class DataModule(LightningDataModule):
-    def __init__(self, fanouts, batch_size, num_workers, dataset):
+    def __init__(self, dataset, fanouts, batch_size, num_workers):
```

Comment:
@yxy235 primary parameters should come first in the parameter list. FYI.

Reply:
Get.

> @yxy235 primary parameters should come first in the parameter list. FYI.